bluepysnap/circuit_validation.py

```diff
@@ -206,6 +207,40 @@ def _get_population_groups(population_h5):
             if isinstance(population_h5[name], h5py.Group) and name.isdigit()]
 
 
+def _sonata_group_to_dataframe(group, types_file, type_id, id_, group_id, group_index):
+    """Transforms hdf5 population group to pandas DataFrame.
+
+    Args:
+        group: HDF5 group of edges or nodes population
```

Comment:
I am pretty sure we cannot do that for edges. 
With a "normal" edge file (few hundred of millions of edges to multiple billions of edges), the data cannot fit in memory.
Did you try with a big circuit ? 

Maybe we should keep that for nodes only at the moment ?

Reply:
Ah ok I saw that you are using it only for the nodes. 
Is it something you plan on using on edges too ?

Reply:
I thought to use it for edges but now I'm thinking. It will definitely fail on few hundred of millions of edges. I will reimplement it only for nodes.